gettsim/pre_processing/policy_for_date.py

```diff
@@ -50,7 +50,7 @@
 from gettsim.taxes.zu_verst_eink.vorsorge import _vorsorge_bis_2004
 
 
-def get_policies_for_date(policy_date, policy_groups="all"):
+def get_policies_for_date(policy_date, policy_groups="all", groups=None):
```

Comment:
Rest is unrelated to piecewise polynomial, right? Necessary in this PR?

Reply:
We changed the interface before to `policy_groups`. I just introduced this to support the old interface. Given the time pressure, I thought, that this little change could be done quickly in here!

Reply:
Ahh, sorry. Just revert. I do not think anybody used that, we certainly did not in class.

I must confess I do not even know what it does exactly, although the new name allows me to guess :grinning:

Reply:
Sorry for that misunderstanding somewhere on Zulip.

Reply:
Entirely my fault for not digging closely enough into what you changed about the interface.

Reply:
Reverted!